styles/itsy.css

```diff
@@ -30,7 +30,11 @@ header {
   font-size: 25px;
 }
 
-.active, .header-menu__item:hover, .header__icon:hover {
+header a {
```

Comment:
what is it doing here? should be in the `header_footer.css`?

Reply:
since here the title is part of the main block, not a stand-alone block.